### ISSUE TITLE
Toggle Krypton extension on and off

### ISSUE THREAD

**Issue by phillipseamore** [kryptco/kr-u2f#6]
Would love it if I could toggle Krypton on and off from the browser-action icon for when I need to use my physical key to login and add Krypton.  It's a bit of a hassle to go and disable and then re-enable the extension each time.

**Comment by rpendleton** [comment]
I agree that some way of disabling the extension or changing its priority would be good.

When my phone is dead or doesn't have service/wifi, the extension seems to just hang and never falls back to my hardware key. If I already have my hardware key plugged in, I'd actually prefer to use that instead of my phone since it's usually quicker and more convenient.
